gateway/api/migrations/0011_jobconfig_job_config.py

```diff
@@ -27,21 +26,19 @@ class Migration(migrations.Migration):
                 ),
                 ("created", models.DateTimeField(auto_now_add=True)),
                 ("auto_scaling", models.BooleanField(default=False, null=True)),
+                ("workers", models.IntegerField(null=True)),
+                ("min_workers", models.IntegerField(null=True)),
+                ("max_workers", models.IntegerField(null=True)),
                 (
-                    "workers",
-                    models.IntegerField(
-                        null=True,
-                    ),
-                ),
-                (
-                    "min_workers",
-                    models.IntegerField(
-                        null=True,
-                    ),
-                ),
-                (
-                    "max_workers",
-                    models.IntegerField(
+                    "python_version",
```

Comment:
I think, this should be a separate migration file, rather than changing existing one

Reply:
OK.  I thought it's better changing the file than adding another file because it's in the same model.  It doesn't matter.  I'll add another migration file.

Reply:
You are right, in this case it is not matter as we will introduce those changes together. But if we were introducing those changes separately, we might bump into a situation when previous migration file is applied and we introduced new changes in this file, that will cause migration conflict.